tools/cold-backup/run.sh

```diff
@@ -0,0 +1,482 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+#set -x #debug
+__TMP="/tmp/strimzi-backup" && mkdir -p $__TMP
+__HOME="" && pushd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" >/dev/null \
+    && { __HOME=$PWD; popd >/dev/null; }
```

Comment:
This does not seem to work for me ... when I run it from any other directory then directly whre the `run.sh` is, it fails to load the multiple platform support.